socorro/external/boto/crash_data.py

```diff
@@ -60,10 +60,7 @@ def get(self, **kwargs):
             else:
                 return get(params["uuid"])
         except CrashIDNotFound as cidnf:
-            self.logger.warning(
-                "%(datatype)s not found: %(exception)s",
-                {"datatype": params["datatype"], "exception": cidnf},
-            )
+            self.logger.warning("%s not found: %s", params["datatype"], cidnf)
```

Comment:
I went through and fixed logging messages. However, now that I'm revisiting the logging docs, this was valid before I changed it. I think I'm going to leave it like this.